tests/manage/pv_services/test_pvc_invalid_inputs.py

```diff
@@ -140,3 +167,45 @@ def create_pvc_invalid_size(pvcsize):
                 "PVC creation with invalid size succeeded : "
                 "NOT expected"
             )
+
+
+def create_pvc_invalid_namespace(pvcsize, namespace=None, volume_mode=None, access_mode=constants.ACCESS_MODE_RWO):
```

Comment:
Same, try utilizing the existing function for pvc creation, and in case you expect it to fail in the test, handle the exception there